test/runtests.jl

```diff
@@ -423,6 +426,7 @@ cd(@__DIR__) do
         Test.push_testset(fake)
         Test.record(o_ts, fake)
         Test.pop_testset()
+        open(write_testset_json, "results.json", "w")
```

Comment:
Should we add `results.json` to `.gitignore`?

Reply:
Probably. I've also spotted this is also not quite right. I'll make both changes now, and remove the attribution too.